website/docs/index.html.markdown

```diff
@@ -111,15 +109,8 @@ The following arguments are supported in the `provider` block:
 
 Note: If you have a PEM file on disk, you can pass it in via `pem_file = file("path/to/file.pem")`.
 
-For backwards compatibility, if more than one of `owner`, `organization`,
-`GITHUB_OWNER` and `GITHUB_ORGANIZATION` are set, the first in this
-list takes priority.
+## Authenticating with multiple providers
 
-1. Setting `organization` in the GitHub provider configuration.
-2. Setting the `GITHUB_ORGANIZATION` environment variable.
-3. Setting the `GITHUB_OWNER` environment variable.
-4. Setting `owner` in the GitHub provider configuration.
+When authenticating with multiple providers, it is possible to fail to authenticate both of them correctly and experience unexpected behavior. This is especially fraught when using both `integrations/github` and `hashicorp/github`. As [this GitHub comment](https://github.com/integrations/terraform-provider-github/issues/655#issuecomment-804419698) from @tibbes details, [local names](https://www.terraform.io/docs/language/providers/requirements.html#local-names) must be used when using two providers with the same name. Otherwise, as @tibbes mentions, the `provider "github"` block is sent to one of the providers but not the other, which means that the other provider is working in anonymous mode with no token, no owner, and no organization.
```

Comment:
@tibbes I completely stole most of this content from your excellent comment [here](https://github.com/integrations/terraform-provider-github/issues/655#issuecomment-804419698) since I think it should be added to our official docs. 

Is this form of attribution okay with you? Would you prefer that I revert this and that you make a commit yourself? Is there another way to go about this? I'm not quite sure what the best path forward is, but I think your comment is amazing and should be on the website, so I'd like to help make that happen somehow.

Reply:
Yes, what you've added is absolutely fine by me!

Reply:
Sounds good, thank you! I'll resolve this comment for now, but please let me know if you change your mind or if you'd like to alter this content at all. Thanks!